presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -205,6 +206,11 @@ public SystemSessionProperties(
                         "Enable optimization for metadata queries",
                         featuresConfig.isOptimizeMetadataQueries(),
                         false),
+                booleanSessionProperty(
+                        OPTIMIZE_NESTED_COLUMNS,
+                        "Enable optimization for nested columns",
```

Comment:
we may want to rename "nested columns" as at some point we also say "nested fields", so we have a mix of terminology that we use. What we do in this patch is to prune unused struct fields so maybe we should use a terminology to reflect that.